plugins/sentinelone/komand_sentinelone/util/helper.py

```diff
@@ -11,3 +15,12 @@ class BlacklistMessage:
     blocked = "The given hash has been blocked"
     unblocked = "The given hash has been unlocked"
     not_exists = "The given hash does not exist"
+
+
+def check_password_meets_requirements(password):
+    if password:
```

Comment:
Can we delete this line since we can safely assume password would be provided if this function is called?